plugins/chain_plugin/tracked_votes.cpp

```diff
@@ -39,9 +39,11 @@ namespace eosio::chain_apis {
                return;
 
             if (!block->contains_extension(chain::quorum_certificate_extension::extension_id())) {
-               fc_ilog(chain::vote_logger, "Block ${id}... #${n} @ ${t} produced by ${p}, latency: ${l}ms has no votes",
+               std::optional<chain::block_header_extension> fin_ext = block->extract_header_extension(chain::finality_extension::extension_id());
+               chain::qc_claim_t claim = fin_ext ? std::get<chain::finality_extension>(*fin_ext).qc_claim : chain::qc_claim_t{};
```

Comment:
Should we extract the header extension only if `vote_logger` is enabled?

Reply:
It is logged at `info` which is default. User would have to set to `warn` or `error` or `off`. But yes, probably should avoid.